config/locales/es.yml

```diff
@@ -321,23 +321,23 @@ es:
     recovery_code_html:
     incorrect_otp: Tu código OTP no es correcto.
     otp_code: código OTP
-    require_mfa_disabled: Se ha activado la autenticación de múltiples factores. Para reconfigurarla, primero tendrás que desactivarla.
+    require_mfa_disabled: Se ha activado la autenticación de múltiples factores. Para
+      reconfigurarla, primero tendrás que desactivarla.
```

Comment:
It seems like the languages are all messed up, with some values being deleted because some keys went up or down a hierarchical level. I'll work on this tomorrow.